files/justfiles/luks.just

```diff
@@ -20,3 +20,8 @@ setup-luks-tpm-unlock:
 remove-luks-tpm-unlock:
     #!/usr/bin/bash
     run0 /usr/libexec/luks-disable-tpm2-autounlock
+
+# Enabled automatic LUKS unlock via a FIDO2 hardware key
+setup-luks-fido2-unlock:
```

Comment:
can we have a remove-luks-fido2-unlock as well?

Reply:
Sure